### PR TITLE
new pipeline for pushing same data for intent, entities & slot/call tagging

### PR DESCRIPTION
**Pull request by greed2411**

### CODE REVIEW

File: skit_pipelines/components/tag_calls/__init__.py [hunk 1]
```diff
@@ -12,6 +12,7 @@ def tag_calls(
     job_ids: str = "",
     project_id: Optional[str] = None,
     org_id: Optional[str] = None,
+    labelstudio_call_project_id : Optional[str] = None,
```
Comment (biswaroop1547) [inline skit_pipelines/components/tag_calls/__init__.py:15]: can we call this `call_project_id`

**Review by biswaroop1547 — CHANGES_REQUESTED** [review]

File: skit_pipelines/components/tag_calls/__init__.py (hunk 1 above)
Comment (greed2411) [reply skit_pipelines/components/tag_calls/__init__.py:15]: want to be explicit about labelstudio man.

Comment (biswaroop1547) [reply skit_pipelines/components/tag_calls/__init__.py:15]: since we ditching tog anyway, our params should be annotation tool agnostic

**Review by biswaroop1547 — CHANGES_REQUESTED** [review]

File: skit_pipelines/components/tag_calls/__init__.py (hunk 1 above)
Comment (greed2411) [reply skit_pipelines/components/tag_calls/__init__.py:15]: lol fine.

**Comment by greed2411** [comment]
have made the necessary changes

**Review by biswaroop1547 — APPROVED** [review]

**Pull request merged**